lxd/util/http.go

```diff
@@ -93,7 +93,7 @@ func EtagCheck(r *http.Request, data any) error {
 	}
 
 	if hash != match {
-		return api.StatusErrorf(http.StatusPreconditionFailed, "ETag doesn't match: %s vs %s", hash, match)
+		return api.StatusErrorf(http.StatusPreconditionFailed, "ETag doesn't match: %s vs %s. The configuration has been modified since this change began. Please reload the configuration before proceeding.", hash, match)
```

Comment:
How about "Please retrieve the updated configuration before proceed" as im worried "reload" here might suggest to the users they need to reload LXD.

Reply:
Yes, good idea as the "reload" was bit UI focused and can be misunderstood on CLI. Adopting to the suggestion.